common/baselines/librispeech/ls100/gmm/baseline_config.py

```diff
@@ -0,0 +1,57 @@
+"""
+Definition of the pipeline in terms of inputs and steps that are executed
+"""
+from sisyphus import gs
+
+from i6_experiments.common.setups.rasr import gmm_system
+from i6_experiments.common.setups.rasr.util import RasrSteps, OutputArgs
+
+from i6_experiments.common.baselines.librispeech.ls100.gmm import baseline_args
+from i6_experiments.common.baselines.librispeech.data import get_corpus_data_inputs
+
+from ...default_tools import RASR_BINARY_PATH
+
+
+def run_librispeech_100_common_baseline(
+    alias_prefix="baselines/librispeech/ls100/gmm/common_baseline",
+):
+
+    gs.ALIAS_AND_OUTPUT_SUBDIR = alias_prefix
+
+    hybrid_init_args = baseline_args.get_init_args()
+    mono_args = baseline_args.get_monophone_args()
+    cart_args = baseline_args.get_cart_args()
```

Comment:
now here is a wrong parameter I think: as you do not give any options to `get_cart_args()`, the returned questions will contain the unknown phoneme which is incompatible with the lexicon returned below using `use_g2p_training=True`